web-ng/src/app/components/feature-panel-header/feature-panel-header.component.html

```diff
@@ -15,10 +15,15 @@
 -->
 
 <mat-list-item>
-  <img matListAvatar [src]="pinUrl" />
-  <h3 matLine>
-    {{ layer?.name?.get(lang) }}
-  </h3>
+  <img *ngIf="featureType === 'point'" matListAvatar [src]="pinUrl" />
+  <ng-container *ngIf="featureType === 'polygon'">
+    Polygon will be added here
```

Comment:
TODO: Couldn't find any material icon?

Reply:
@jacobmclaws Can you please share the polygon icon you used in your mocks?

Reply:
Are we talking about this one? ![image](https://user-images.githubusercontent.com/9397366/122880715-45c36600-d375-11eb-98b2-96ca11c424f2.png)

Here's a folder with png and svg version: 
[Ground Web App (12).zip](https://github.com/google/ground-platform/files/6692018/Ground.Web.App.12.zip)

Reply:
That's the one, thank you! @HemanParbhakar over to you!